portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,59 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['I\'m ready, I\'m ready, I\'m ready...', 'You know what\'s funnier than 24? 25!', 'East?! I thought you said Weest!', 'Squidwaard, I used your clarinet to unclog my toilet!'];
+      ['I\'m ready, I\'m ready, I\'m ready...', 
+      'You know what\'s funnier than 24? 25!', 
+      'East?! I thought you said Weest!', 
+      'Squidwaard, I used your clarinet to unclog my toilet!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+}
+
+/**
+ * Reveal hidden description upon mouseover and hides the project title
+   text.
+ * @param {object} hoveredItem An anchor tag containing the
+      project image and text
```

Comment:
Here and elsewhere. Comments should be written as sentences, ending with periods.

see: https://developers.google.com/style/periods

Reply:
I converted the inline comments to be sentences as well. For the param description, I only added a period at the end.

Done.